gateway/ln-gateway/src/lib.rs

```diff
@@ -1709,7 +1749,7 @@ impl Gateway {
         let routing_fees = gateway_parameters
             .fees
             .clone()
-            .unwrap_or(GatewayFee(PaymentFee::SEND_FEE_DEFAULT.into()));
+            .unwrap_or(GatewayFee(PaymentFee::TRANSACTION_FEE_DEFAULT.into()));
```

Comment:
Wrapping this in a GatewayFee here seems unnecessary.

Reply:
Fixed